airbyte-integrations/connectors/source-monday/source_monday/item_pagination_strategy.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Mapping, Optional, List, Tuple
+
+from airbyte_cdk.sources.declarative.requesters.paginators.strategies.page_increment import PageIncrement
+
+
+class ItemPaginationStrategy(PageIncrement):
+    """
+    Page increment strategy with subpages for the `items` stream.
+
+    See Monday API documentation https://developer.monday.com/api-reference/docs.
```

Comment:
Can you reference a better location where future developers can find information to this specific custom component?

Reply:
I can put something like this:

From the `items` documentation https://developer.monday.com/api-reference/docs/items
> Please note that you cannot return more than 100 items per query when using items at the root. To [adjust](https://developer.monday.com/api-reference/changelog/new-limits-to-the-items-query-in-90-days) your query, try only returning items on a specific board, nesting items inside a boards query, looping through the boards on your account, or querying less than 100 items at a time.

This pagination strategy supports nested loop through `boards` on the top level and `items` on the second. See `boards` documentation for more details: https://developer.monday.com/api-reference/docs/boards#queries